lib/constants/profile_constants.dart

```diff
@@ -0,0 +1,17 @@
+// Widget dimensions
+
+const double kProfileHeaderHeight = 160.0;
+
+// Section titles
+
+const String SECTION_TITLE_SUMMARY = "Summary";
+const String SECTION_TITLE_EDUCATION = "Education";
+const String SECTION_TITLE_EXPERIENCE = "Experience";
+const String SECTION_TITLE_SKILLS = "Skills";
+const String SECTION_TITLE_CONTACT = "Contact";
+
+// Contact info
+
+const String CONTACT_EMAIL_ADDRESS = "Email address";
+const String CONTACT_PHONE_NUMBER = "Phone number";
+const String CONTACT_WEBSITE = "Website";
```

Comment:
Same in here. Used this sample code snippet as a model: https://www.driftycode.com/flutter-global-constants-for-the-complete-project/